File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/util/Navigation.java
package io.casey.musikcube.remote.util;

import android.app.Activity;

public final class Navigation {
    private Navigation() {
    }

    public interface RequestCode {
        int ALBUM_TRACKS_ACTIVITY = 10;
        int ALBUM_BROWSE_ACTIVITY = 11;
        int CATEGORY_TRACKS_ACTIVITY = 12;
    }

    public interface ResponseCode {
        int PLAYBACK_STARTED = Activity.RESULT_FIRST_USER + 0xbeef;
    }
}

File: src/musikdroid/app/src/main/java/io/casey/musikcube/remote/ui/shared/constants/Navigation.kt
package io.casey.musikcube.remote.ui.shared.constants

import android.app.Activity

object Navigation {
    interface RequestCode {
        companion object {
            val ALBUM_TRACKS_ACTIVITY = 10
            val ALBUM_BROWSE_ACTIVITY = 11
            val CATEGORY_TRACKS_ACTIVITY = 12
        }
    }

    interface ResponseCode {
        companion object {
            val PLAYBACK_STARTED = Activity.RESULT_FIRST_USER + 0xbeef
        }
    }
}
